server/src/main/java/com/lunchtime/models/Feedback.java

```diff
@@ -0,0 +1,62 @@
+package com.lunchtime.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Setter;
+import lombok.Getter;
+import org.hibernate.annotations.ColumnDefault;
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.sql.Date;
+
+@Entity
+@Setter
+@Getter
+public class Feedback {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @NotNull
+    @Column(name = "description", columnDefinition = "TEXT")
+    private String description;
+
+    @ColumnDefault("true")
+    @Column(name = "is_active")
+    private Boolean isActive;
+
+    @NotNull
+    @Column(name = "date")
+    private Date date;
+
+    @NotNull
+    @Column(name = "user_name")
+    private String userName;
+
+    @NotNull
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JsonIgnore
+    @JoinColumn(name = "rest_id",referencedColumnName = "id")
+    private Restaurant restId;
+
+    @ColumnDefault("0")
+    @Column(name = "counter_like")
+    private Integer counterLike;
+
+    @ColumnDefault("0")
+    @Column(name = "counter_dislike")
+    private Integer counterDislike;
+
+    public Feedback() {   }
+
+    public Feedback(String description, Boolean isActive, Date date,
+                    String userName, Restaurant restId, Integer counterLike, Integer counterDislike) {
+        this.description = description;
+        this.isActive = isActive;
+        this.date = date;
+        this.userName = userName;
+        this.restId = restId;
+        this.counterLike = counterLike;
+        this.counterDislike = counterDislike;
+    }
```

Comment:
As mentioned before, we can use **[@NoArgsConstructor](https://projectlombok.org/api/lombok/NoArgsConstructor.html)** and **[@AllArgsConstructor](https://projectlombok.org/api/lombok/AllArgsConstructor.html)** here. Here is a great **[article](http://stevenmwesigwa.com/tutorials/project-lombok/8/how-to-use-allargsconstructor-annotation-with-project-lombok-in-java-applicat)** about `@AllArgsConstructor` usage practice.

Reply:
https://www.baeldung.com/hibernate-identifiers
id not initialize

Reply:
Give me please correct exact line what is needed for your comment.

Reply:
.../models/feedback
line 15, 16

Reply:
Bro I really can find `@Id` in code but I'm about your link. What exactly you wanted me to see **[here](https://www.baeldung.com/hibernate-identifiers)**?

Reply:
Sorru, I missed `@Id` that is `@GeneratedValue`. **[Here](https://stackoverflow.com/questions/48784923/is-using-id-field-in-allargsconstructor-while-using-spring-jpa-correct)** it is recommended to use base class and inherit it. I think this can be a great solution for whole project entities..